src/platform/plugins/shared/dashboard/public/panel_placement/place_new_panel_strategies.ts

```diff
@@ -15,15 +15,18 @@ import { PanelPlacementProps, PanelPlacementReturn } from './types';
 
 export const runPanelPlacementStrategy = (
   strategy: PanelPlacementStrategy,
-  { width, height, currentPanels }: PanelPlacementProps
+  { width, height, currentPanels, sectionId }: PanelPlacementProps
 ): PanelPlacementReturn => {
   switch (strategy) {
     case PanelPlacementStrategy.placeAtTop:
       const otherPanels = { ...currentPanels };
       for (const [id, panel] of Object.entries(currentPanels)) {
-        const { gridData, ...currentPanel } = cloneDeep(panel);
-        const newGridData = { ...gridData, y: gridData.y + height };
-        otherPanels[id] = { ...currentPanel, gridData: newGridData };
+        // only consider collisions with panels in the same section
+        if (panel.gridData.sectionId === sectionId) {
```

Comment:
How about adding some unit tests for collapsable sections?

Reply:
Good call - added tests in 219bec8c300ec4e05e37b54a487c9b8e5d6d160d